src/main/java/com/education/repository/TimetableRepositoryImpl.java

```diff
@@ -1,17 +1,21 @@
 package com.education.repository;
 
+import com.education.domain.Student;
 import com.education.domain.Timetable;
+import com.education.service.StudentService;
 import org.hibernate.Session;
 import org.hibernate.SessionFactory;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.time.LocalDate;
 import java.time.LocalTime;
+import java.util.ArrayList;
 import java.util.List;
 
 @Transactional
 public class TimetableRepositoryImpl implements TimetableRepository {
     private SessionFactory sessionFactory;
+    private StudentService studentService;
```

Comment:
You should remove this line for two reasons:
- It is not used
- It's not a good idea to talk with the service layer from the repository